xclim/indicators/atmos/_temperature.py

```diff
@@ -1258,3 +1260,45 @@ def cfcheck(self, tas, snd=None):
     compute=indices.latitude_temperature_index,
     parameters={"lat": {"kind": InputKind.VARIABLE}, "lat_factor": 60},
 )
+
+
+australian_hardiness_zones = Temp(
+    title="Australian hardiness zones",
+    identifier="australian_hardiness_zones",
+    units="",
+    long_name="Australian National Botanic Gardens scale of average annual minimal temperatures over given time period",
+    description="A climate indice based on a rolling average of minimal annual temperature of {window} years. "
```

Comment:
```suggestion
    description="A climate indice based on a {window} years rolling average of the annual minimum temperature . "
```

Reply:
To validate to make sure the formulation is understood.

Reply:
See note about the averaging window for method `anbg`.